tst/testextra.g

```diff
@@ -0,0 +1,26 @@
+#############################################################################
+##
+##  This file is part of GAP, a system for computational discrete algebra.
+##
+##  Copyright of GAP belongs to its developers, whose names are too numerous
+##  to list here. Please refer to the COPYRIGHT file for details.
+##
+##  SPDX-License-Identifier: GPL-2.0-or-later
+##
+##  This file exercices 'tst/testextra' tests, which are very slow. 
+##
+##  To run it, use this command:
+##
+##  gap> Read( Filename( DirectoriesLibrary( "tst" ), "testextra.g" ) );
+##
+
+Print( "You should start GAP4 using `gap -A -x 80 -r -m 100m -o 1g -K 2g'.\n\n" );
```

Comment:
Do we really want to give that recommendation? Why? Why in particular the `-x 80` (the tests automatically restrict the screen size)?

Reply:
This is out of the scope of this PR - this recommendation is in all test files.

Perhaps it means "If you want to run the test locally and see the same line wrapping and sme behaviour as when you start it with make test{something}, start GAP with this screen size and other settings".